.github/workflows/docker.yml

```diff
@@ -3,21 +3,55 @@ name: Publish Docker image
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
-  create:
+  push:
+    branches:
+      - develop
+    tags:
+      - "v*"
 
 jobs:
   push_to_registry:
-    if: ${{ startsWith(github.ref, 'refs/tags/v') }}
     name: Push Docker image to GitHub Packages
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-20.04"
     steps:
-      - name: Check out the repo
+      - name: Check out the repo of ${{ github.repository }}
         uses: actions/checkout@v2
-      - name: Push to GitHub Registry
-        uses: mr-smithers-excellent/docker-build-push@v5
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_IO_USER }}
+          password: ${{ secrets.DOCKER_IO_ACCESS_TOKEN }}
+
+      - name: Build and push to DockerHub
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          tags: "${{ secrets.DOCKER_IO_USER }}/cobrexa.jl:latest"
```

Comment:
can we somewhat make `cobrexa.jl` as a variable name?

Reply:
done, we can actually use another metadata extraction action to make this much better (also with tags)